app/src/main/contrib/components/Responsibilities/Overview/ResponsibilityOverviewPage.tsx

```diff
@@ -31,55 +32,21 @@ export class ResponsibilityOverviewPage extends ContribPageWithHeader<LocationPr
     }
 
     title() {
-        return <ResponsibilityOverviewTitle />
+        return <ResponsibilityOverviewTitle />;
     }
 
     urlFragment(): string {
         const s = responsibilityStore.getState();
-        return `responsibilities/${s.currentTouchstone.id}/`;
+        return `responsibilities/${s.currentTouchstone ? s.currentTouchstone.id : ''}/`;
```

Comment:
I don't think this is needed and in any case the path `/responsibilities/` would result in a 404 so there'd be no point directing users to that! This function will only get called either on this page after `super.load()` is called (by which time `s.currentTouchstone` will be set) or if this page was a parent for another page (which I don't think it is right?)